weather.py

```diff
@@ -35,7 +37,10 @@ def convert_f_to_c(temp_in_farenheit):
     Returns:
         A float representing a temperature in degrees celcius, rounded to 1dp.
     """
-    pass
+    temp_in_farenheit=float(temp_in_farenheit)
+    result=((temp_in_farenheit-32)*5)/9
+    result=round(result,1)
```

Comment:
It's generally good practice to store updated values in separate variables, rather than use intermediate/temp working variables. This mainly helps with code readability e.g.
```suggestion
    float_temp_in_farenheit=float(temp_in_farenheit)
    temp_in_celsius=((float_temp_in_farenheit-32)*5)/9
    result=round(result,1)
```